index.js

```diff
@@ -210,9 +242,16 @@ async function underPressure (fastify, opts) {
         throw underPressureError
       }
     }
+    if (req.isUnderPressureError) {
+      return { status: 'not-ok' }
```

Comment:
Why did you add this?

Reply:
As I described in the PR description:

Both routes /status and /status/memory-usage will always return a value even if the server is under pressure.
Route /status returns not-ok when server is under pressure.

Reply:
This will make under pressure return a 200 even if the status is not ok. I don't think this is correct as load balancers do not really parse this JSON, but they rather rely on http status codes, and throwing did this for us.

Reply:
My original idea was to return a non OK status along side the reason causing the problem (ie event loop delay, heap usage, rss usage, external health check). Didn't implement it at the end, but I would find it useful. Nothing would stop it from being a 503 instead of a 200.

Reply:
I would prefer it to keep the previous behavior in this PR and just expose the new route. I would also remove the `req[isUnderPressureError]` property as it should not be needed anymore. If you'd like to add a parsable non-ok status, let's do it in a different pR.